pageserver/compaction/src/identify_levels.rs

```diff
@@ -308,6 +309,26 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn repro_identify_levels_bails_too_ealy_if_partitioned_keyspace_same_lsn() -> anyhow::Result<()> {
+        tracing_subscriber::fmt::init(); // so that RUST_LOG=trace works
```

Comment:
This will fail if there is multiple tests using it because you can set it only once per process.

Instead, I suggest copy-pasting from tests.rs:

```Rust
static LOG_HANDLE: OnceCell<()> = OnceCell::new();

pub(crate) fn setup_logging() {
    LOG_HANDLE.get_or_init(|| {
        logging::init(
            logging::LogFormat::Test,
            logging::TracingErrorLayerEnablement::EnableWithRustLogFilter,
            logging::Output::Stdout,
        )
        .expect("Failed to init test logging")
    });
}
```